assets/script/script.js

```diff
@@ -0,0 +1,20 @@
+const CloseButton = document.getElementById('icone_container');
+const HumbergerButton = document.getElementById('mob_menu');
+const MobileMenu = document.getElementById('mobile_menu');
+const PortfolioButton = document.getElementById('portfolio_button');
+const AboutButton = document.getElementById('about_button');
+const ContactButton = document.getElementById('contact_buttton');
+
+function close() {
+  MobileMenu.style.display = 'none';
+}
+
+function open() {
+  MobileMenu.style.display = 'block';
+}
+
+CloseButton.addEventListener('click', close);
+HumbergerButton.addEventListener('click', open);
+PortfolioButton.addEventListener('click', close);
+AboutButton.addEventListener('click', close);
```

Comment:
- [ ] Great job working on the functionality for the mobile menu it is working fine but I have small note on the design it could be improved, I think it would be great if you add a blur on the background so the user will not be able to see the content behind the menu so kindly add mote blur to match the Figma design as much as posible 💯✅🚀

- [ ] I think it would be great if you remove the scroll in the side while the menu is open since it shouldn't be there so you can add this property `overflow-Y:  hidden;` so that would prevent the scrolling because that bad for user expereince to scrol while the menu is open since it shouldn't be any scrolling in the side ✅💯💪🏻🚀

- Your design
![image](https://github.com/WilfriedM01/portfolio/assets/58553711/7f6ba437-0bd1-4937-941e-8a72160ca6a8)

- Figma 
![image](https://github.com/WilfriedM01/portfolio/assets/58553711/ea2ca9a1-e018-4461-9e6c-e55739bdc77c)